src/css/vendor/_overwrites_uswds.scss

```diff
@@ -4,45 +4,108 @@ h3,
 h4,
 h5,
 h6 {
-  font-family: $font-sans;
-  font-weight: $font-medium;
+  font-family: $coa-font-sans;
+  font-weight: $coa-font-extrabold;
   margin: 0;
 }
 
-ol, ul {
-  margin: 0;
-  padding: 0;
-  list-style-position: inside;
-}
+h1 {
+  font-size: $coa-font-size-medium;
+  line-height: $coa-line-height-medium;
+  margin-bottom: $coa-spacing-small;
+  margin-top: $coa-spacing-medium;
 
-label, textarea, input, select {
-  max-width: none;
+  @include media($coa-small-screen) {
+    font-size: $coa-font-size-large;
+    line-height: $coa-line-height-large;
+    margin-bottom: $coa-spacing-medium;
+    margin-top: $coa-spacing-large;
+  }
+
+  @include media($coa-medium-screen) {
+    font-size: $coa-font-size-xlarge;
+    line-height: $coa-line-height-xlarge;
+    margin-bottom: $coa-spacing-large;
+    margin-top: $coa-spacing-xlarge;
+  }
+
+  @include media($coa-large-screen) {
+    font-size: $coa-font-size-xxlarge;
+    line-height: $coa-line-height-xxlarge;
+    margin-bottom: $coa-spacing-large;
+    margin-top: $coa-spacing-xlarge;
+  }
 }
 
-.usa-table-borderless {
-  margin: 0;
-  padding: 0;
+h2 {
+  font-size: $coa-font-size-small;
+  line-height: $coa-line-height-small;
+  margin-bottom: $coa-spacing-small;
+  margin-top: $coa-spacing-medium;
 
-  td, th {
-    border: none;
-    margin: 0;
-    padding: 0;
+  @include media($coa-small-screen) {
+    font-size: $coa-font-size-medium;
+    line-height: $coa-line-height-medium;
+    margin-bottom: $coa-spacing-medium;
+    margin-top: $coa-spacing-large;
+  }
+
+  @include media($coa-medium-screen) {
+    font-size: $coa-font-size-large;
+    line-height: $coa-line-height-large;
+    margin-bottom: $coa-spacing-large;
+    margin-top: $coa-spacing-xlarge;
   }
 
-  th:first-child {
-    padding-right: $coa-space-level2;
+  @include media($coa-large-screen) {
+    font-size: $coa-font-size-xlarge;
+    line-height: $coa-line-height-xlarge;
+    margin-bottom: $coa-spacing-large;
+    margin-top: $coa-spacing-xlarge;
   }
 }
 
-.usa-input-error {
-  margin-top: 1rem;
-  margin-bottom: 1rem;
+ol, ul {
+  margin: 0;
+  padding: 0;
 
-  label {
-    width: calc(100% + 1.9rem); // 1.5rem left padding + 4px border from input error spacing
+  li {
+    margin: 0;
   }
 }
 
-.usa-input-error-message {
-  margin-bottom: $coa-space-level2;
+p {
+  margin: 0;
 }
+
+// label, textarea, input, select {
```

Comment:
should we delete these instead of comment them out?

Reply:
Yea, I've mostly left them there since I needed them before when I was building out the service page and suspected I would need them again to overwrite the uswds stuff. I've been uncommenting as I go. But I can delete the ones we definitely don't end up using.